maple-sdk/introduction.md

```diff
@@ -0,0 +1,49 @@
+# Introduction
+
+`maple-js` is a powerful and user-friendly JavaScript SDK designed for seamless interaction with Maple Protocol's smart contracts on the Ethereum blockchain. This SDK provides developers with essential tools to easily integrate with the Maple Protocol ecosystem, streamlining the process of connecting to the blockchain, managing accounts, submitting transactions, and handling events generated by smart contracts.
+
+By utilizing the JavaScript SDK for Maple Protocol, developers can efficiently build robust and secure dApps that unlock the full potential of Maple Protocol's smart contracts, encouraging innovation and promoting the adoption of decentralized finance solutions.
+
+Whether you're developing backend services, frontend applications, or a combination of both, the Maple SDK offers the tools necessary for a smooth user experience.
+
+**TypeChain Exports:**
+
+`maple-js` exports are provided via TypeChain - convenient and error-resistant way for developers to call smart contract functions, ensuring a smoother integration with Maple Protocol's ecosystem.
+
+TypeChain exports are automatically generated TypeScript bindings for Ethereum smart contracts. These bindings make it easier for developers to interact with smart contracts by providing type-safe methods for contract functions. This means that developers can leverage TypeScript's type-checking and autocomplete features, reducing the likelihood of errors and improving the overall development experience.
+
+# Exports
+
+| Export                 | Purpose                                                                                 |
+| ---------------------- | --------------------------------------------------------------------------------------- |
+| `addresses`            | Contains the contract addresses for different networks (mainnet, testnet)               |
+| `bPool`                | Manages Balancer liquidity pool functionality                                           |
+| `chainlink`            | Manages Chainlink oracle functionality                                                  |
```

Comment:
not sure we want to include this ... users can RPC maple globals oracles to get price rather than chainlink directly